app/controllers/patients_controller.rb

```diff
@@ -848,8 +848,10 @@ def current_case_status
 
     patient_ids = params[:patient_ids]
     patients = current_user.viewable_patients.where(id: patient_ids)
+    unique_pha = patients.distinct.pluck(:public_health_action)
+    pui_elidgible = !(unique_pha.empty? || (unique_pha.length == 1 && unique_pha.include?('None')))
```

Comment:
```suggestion
    pui_eligible = !(unique_pha.empty? || (unique_pha.length == 1 && unique_pha.include?('None')))
```

Reply:
Thanks for catching this [1c361a9](https://github.com/SaraAlert/SaraAlert/pull/1242/commits/1c361a9abce8255de86007fb083440196b257123)